libs/core-ui/src/lib/Highchart/getDefaultHighchartOptions.ts

```diff
@@ -34,20 +34,6 @@ export function getDefaultHighchartOptions(theme: ITheme): Highcharts.Options {
       zoomType: "xy"
     },
     credits: undefined,
-    exporting: {
```

Comment:
We use default exporting which includes all options below as well as downloadSVG